administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -7,9 +7,24 @@ COM_JOOMLAUPDATE_CAPTIVE_HEADLINE="Confirm your credentials"
 COM_JOOMLAUPDATE_CHECKED_UPDATES="Checked for updates."
 COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_DESC="Shows the checkbox to confirm you have taken a backup and you're ready to update in the final step before the update is actually applied."
 COM_JOOMLAUPDATE_CONFIG_BACKUPCHECK_LABEL="Confirm Backup Checkbox"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_DESC="How much of the update ZIP archive to download on each page load. Higher values are faster, but may time out on slow servers. Higher values are safer from timeouts, but the update downloads slower. If the download hangs try using 1 MiB or lower values."
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_LABEL="Download size per page load"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_256KB="256 KiB"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_512KB="512 KiB"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_1MIB="1 MiB"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_2MIB="2 MiB"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_5MIB="5 MiB"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_10MIB="10 MiB"
+COM_JOOMLAUPDATE_CONFIG_CHUNK_LENGTH_20MIB="20 MiB"
+COM_JOOMLAUPDATE_CONFIG_CHUNKED_DOWNLOAD_DESC="Set to Yes to download the update packages 1MB at a time, preventing server timeouts on extremely slow servers. Set to No (default) to download the entire update package in one page load; it's faster overall on most servers."
```

```suggestion
COM_JOOMLAUPDATE_CONFIG_CHUNKED_DOWNLOAD_DESC="Set to Yes to download the update packages 1MB at a time, preventing server timeouts on extremely slow servers. Set to No to download the entire update package in one page load; it's faster overall on most servers."
```
I think that's a remainder from before changing the default value in the XML to "1" = "Yes".